docs/installation.md

```diff
@@ -0,0 +1,103 @@
+# Installation
+
+## Deploy the Gateway
+
+You can deploy NGINX Kubernetes Gateway on an existing Kubernetes 1.16+ cluster. The following instructions walk through the steps for deploying on a [kind](https://kind.sigs.k8s.io/) cluster.
+
+1. Load the NGINX Kubernetes Gateway image onto your kind cluster:
+
+   ```
+   kind load docker-image nginx-kubernetes-gateway:0.0.1
+   ```
+
+   Make sure to substitute the image name with the name of the image you built.
+
+1. Install the Gateway CRDs:
+
+   ```
+   kubectl apply -k "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.5.0"
+   ```
+
+1. Create the nginx-gateway namespace:
```

Comment:
Thoughts on a warning here that the namespace MUST be nginx-gateway for the time being?

Reply:
Because that namespace used in the manifests and in the further instructions, I think it is clear that the user should use that namespace. However, nothing prevents them to update all the manifests and instructions to use a different namespace though.

Reply:
If they change it I think the validation on main.go:49 will fail until that bug has been fixed.

Reply:
while the value of that argument requires `nginx-gateway` in it, the validation doesn't check that the Gateway is running in `nginx-gateway` namespace.

Reply:
Sure, but wouldn't a user do this.

Configure everything in `gateway-test`.
Follow the `gateway-ctlr-name` output (DOMAIN/NAMESPACE/NAME) and use `k8s-gateway.nginx.org/gateway-test/foo`.
Fail validation.

```
./build/.out/gateway --gateway-ctlr-name k8s-gateway.nginx.org/gateway-test/foo
failed validation - flag: '--gateway-ctlr-name' reason: 'cross namespace unsupported: gateway-test'
```

My fear is that without a warning, to either keep everything in nginx-gateway or ignore the NAMESPACE and always use nginx-gateway regardless of namespace, we'll induce confusion.

Reply:
ok. I added a warning.